README.md

```diff
@@ -25,8 +25,8 @@ Program to plot histograms in python.
 ## 8. TreeVisualization
 Program to prepare tree visualization of data in python giving output in pdf format.
 
-## 9. RockPaperScissorGame
-Program to let users play Rock Paper Scissor with computer.
+## 9. Rock-Paper-ScissorGame
```

Comment:
We are presently following camel case in the README, so either change all the heading or just leave it as it is.